constants/Bingo.json

```
@@ -1,347 +1,351 @@
+        "6 visitors total at a time (5 queue + 1) without you",
+        "needing to be online."
       ]
     },
-    "Warrior": {
-      "difficulty": "Medium",
+    "Who needs friends?": {
+      "difficulty": "Hard",
       "note": [
-        "Reach Combat 18"
+        "Solo a f1 with 270+"
       ],
       "guide": [
-        "You should either kill Endermen in",
-        "End using your Void Sword (200k) and",
-        "Ender Armor for this (buy the Ender Training",
-        "Perk from Tyzzo next to the Draconic Altar)",
-        "or Spiders in the Spiders Den with a Cleave 5",
-        "weapon. You might want to also use the free",
-        "Combat XP from Bestiary Miletones for this."
+        "This will heavily rely on your gear and Catacombs Level.",
+        "Ideally you'll want to buy the best tier of Goggles from",
+        "Ophelia that you can. Use End Armor to the remaining 3 pieces",
+        "but replace them with better Dungeon Armor drops.",
+        "Reforge your Armor to Wise or Necrotic if you can",
+        "and your Dreadlord Sword to Heroic. Star your Dungeon Gear.",
+        "You should also use the Power Stone that grants the most",
+        "Intelligence that you have access to.",
+        "§cDon't forget to use your Class Abilities!"
       ]
     },
-    "Fashionable": {
+    "Bayou Bobbin'": {
       "difficulty": "Medium",
       "note": [
-        "7 Armor Sets"
+        "4/4 Bayou Armor"
       ],
       "guide": [
-        "Craft Leather, Iron, Gold and Diamond Armor",
-        "Collect Ender Armor from The End (8 pieces)",
-        "Craft Leaflet, Farm Suit, Mushroom and Cactus Armor."
+        "For this you'll need to fish in the Backwater Bayou.",
+        "This requires Angler Armor (bought in The Hub) and",
+        "48 Rusty Coins (Junk catch that requires the Junk Sinker",
+        "from Joel to be applied to your rod)"
       ]
     },
-    "Bone Collector": {
-      "difficulty": "Hard",
-      "note": [
-        "47.5k Bone Collection"
-      ],
-      "guide": [
-        "Kill Skeletons in The Deep Caverns",
-        "to unlock Skeleton Minions. Craft as many",
-        "as you can and place them on your island.",
-        "Give them the best fuel and storage",
-        "that you have access to and collect in time.",
-        "Don't forget to buy your daily bones from",
-        "The Adventurer in The Hub."
-      ]
-    },
-    "Slayer": {
-      "difficulty": "Easy",
+    "They're Prrrrrretty good": {
+      "difficulty": "Medium",
       "note": [
-        "Reach Slayer Level 2"
+        "Consume 4 Dwarven Cereals"
       ],
       "guide": [
-        "Recommended for this is Zombie Slayer.",
-        "This needs 15 xp, so 3 t1 bosses.",
-        "Don't forget to claim the rewards",
-        "from Maddox."
+        "Those have a 20% chance to drop if you participate",
+        "in Active Dwarven Mines Events such as Goblin Raid,",
+        "Raffle or Mithril Gourmand."
```

```suggestion
        "Raffle or Mithril Gourmand.",
        "The chance goes up to 40% if you get the personal bonus."
```
(You may want to double check my wrapping, I'm on mobile.)